includes/data-stores/class-wc-admin-reports-categories-data-store.php

```diff
@@ -239,11 +252,28 @@ public function get_data( $query_args ) {
 				'page_no' => 0,
 			);
 
-			$selections       = $this->selected_columns( $query_args );
-			$sql_query_params = $this->get_sql_query_params( $query_args );
+			$selections          = $this->selected_columns( $query_args );
+			$sql_query_params    = $this->get_sql_query_params( $query_args );
+			$included_categories = $this->get_included_categories_array( $query_args );
+
+			if ( count( $included_categories ) > 0 ) {
+				$fields          = $this->get_fields( $query_args );
+				$join_selections = $this->format_join_selections( array_merge( array( 'category_id' ), $fields ), 'category_id' );
+				$ids_table       = $this->get_ids_table( $included_categories, 'category_id' );
```

Comment:
`get_ids_table` accepts only 1 argument. I see other uses also have 2 🤷‍♂️ .

Reply:
Thanks for taking another look @psealock! This is fixed in 92f6c189995e33384d523788e3524d5c1d5ef069.